DFC.ServiceTaxonomy.ApiFunction/Function/Execute.cs

```diff
@@ -52,6 +52,15 @@ public Execute(IOptionsMonitor<ServiceTaxonomyApiSettings> serviceTaxonomyApiSet
                 var environment = Environment.GetEnvironmentVariable("AZURE_FUNCTIONS_ENVIRONMENT");
                 log.LogInformation($"Function has been triggered in {environment} environment.");
 
+                var host = req.Headers["X-Forwarded-Host"].ToString();
+
+                if (string.IsNullOrWhiteSpace(host))
+                    throw ApiFunctionException.InternalServerError("X-Forwarded-Host header not present.");
+
+                var hostUriBuilder = new UriBuilder { Host = host, Scheme = "http://" };
```

Comment:
I think it might need 'Execute' somewhere in there as well.
Also, we probably shouldn't hard-code http.
It's a bit tricky to handle this generically. We can't use the scheme from the context, because it's the scheme used by APIM that's important.
Plus, even if we had the scheme piped through from APIM in a header, we might be using the host to point to a different endpoint, that could potentially use a different scheme than the current endpoint.
I'd imagine that all the endpoints in an env, will in reality share the same scheme, so perhaps we just have it as a config value and revisit if necessary.